.github/workflows/ci.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       - uses: ./.github/actions/upload-coverage
 
   test-clean-install:
-    needs: [lint-flake8, lint-black-isort]
+    needs: [lint-flake8]
```

Comment:
jobs from [`ci-required.yml`](https://github.com/modin-project/modin/blob/master/.github/workflows/ci-required.yml) are not technically required to run jobs from `ci.yml`, so this PR actually changes the ci's flow, allowing for the jobs to be run even if `lint-black-isort` haven't passed successful (hope it's not a problem, or anyone have a solution for this?)

Reply:
We could only move `isort` to another ci flow, and leave black here. The only reason to leave black is it can find syntax errors and return an error sooner. However I don't mind trying the current solution.

Reply:
ok